modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -155,6 +172,19 @@ The following files are generated in the directory:
 └── worker.ign
 ----
 
+ifdef::osp[]
+. For use in other procedures, export the metadata file's `infraID` key as an environment variable:
++
+----
+$ export INFRA_ID=$(jq -r .infraID metadata.json)
+----
+endif::osp[]
+
+ifdef::osp[]
+[TIP]
+Extract the `infraID` key from `metadata.json` and use it as a prefix for all of the {rh-openstack} resources that you create. By doing so, you avoid name conflicts when making multiple deployments in the same project.
```

Comment:
The import playbook `common.yaml` will automatically parse that information from `metadata.json`; if you think it's better to let the user extract it, then we have to change the code

Reply:
I'd prefer to stick with the playbooks for now, so I'll comment this out as a note to posterity.

Reply:
this is where the playbook willlook for the metadata.json file. It must be in the same directory as the common.yaml playbook, which probably will not be the same directory the openshift-install command create the metadat.json file in.

Reply:
so user has to copy it over.

Reply:
@adduarte Would it make sense to note that in a comment within common.yaml?